yt_dlp/extractor/bluesky.py

```diff
@@ -0,0 +1,221 @@
+            'uploader_url': 'https://bsky.app/profile/clockworkbanana.fun',
+            'channel_url': 'https://bsky.app/profile/did:plc:3tndo2mqg2vgpxnpyrxiol6p',
+            'timestamp': 1726353835,
+            'like_count': int,
+            'repost_count': int,
+            'comment_count': int,
+            'webpage_url': 'https://bsky.app/profile/did:plc:3tndo2mqg2vgpxnpyrxiol6p/post/3l45kdlktfe2o',
+            'tags': 'count:1',
+            'subtitles': dict,
+            'comments': list,
+        },
+    }, {
+        'url': 'https://bsky.app/profile/souris.moe/post/3l4qhp7bcs52c',
+        'md5': '5f2df8c200b5633eb7fb2c984d29772f',
+        'info_dict': {
+            'id': '3l4qhp7bcs52c',
+            'ext': 'mp4',
+            'title': 'maeve: ""',
+            'upload_date': '20240922',
+            'description': '',
+            'thumbnail': r're:^https://video.bsky.app/watch/.*\.jpg$',
+            'alt_title': None,
+            'uploader': 'maeve',
+            'channel': 'souris.moe',
+            'uploader_id': 'did:plc:tj7g244gl5v6ai6cm4f4wlqp',
+            'channel_id': 'did:plc:tj7g244gl5v6ai6cm4f4wlqp',
+            'uploader_url': 'https://bsky.app/profile/souris.moe',
+            'channel_url': 'https://bsky.app/profile/did:plc:tj7g244gl5v6ai6cm4f4wlqp',
+            'timestamp': 1727003838,
+            'like_count': int,
+            'repost_count': int,
+            'comment_count': int,
+            'webpage_url': 'https://bsky.app/profile/souris.moe/post/3l4qhp7bcs52c',
+            'tags': 'count:1',
+            'subtitles': 'count:0',
+            'comments': list,
+        },
+    }, {
+        'url': 'https://bsky.app/profile/de1.pds.tentacle.expert/post/3l3w4tnezek2e',
+        'md5': '1af9c7fda061cf7593bbffca89e43d1c',
+        'info_dict': {
+            'id': '3l3w4tnezek2e',
+            'ext': 'mp4',
+            'title': 'clean: ""',
+            'upload_date': '20240911',
+            'description': '',
+            'thumbnail': r're:^https://video.bsky.app/watch/.*\.jpg$',
+            'alt_title': None,
+            'uploader': 'clean',
+            'channel': 'de1.pds.tentacle.expert',
+            'uploader_id': 'did:web:de1.tentacle.expert',
+            'channel_id': 'did:web:de1.tentacle.expert',
+            'uploader_url': 'https://bsky.app/profile/de1.pds.tentacle.expert',
+            'channel_url': 'https://bsky.app/profile/did:web:de1.tentacle.expert',
+            'timestamp': 1726098823,
+            'like_count': int,
+            'repost_count': int,
+            'comment_count': int,
+            'webpage_url': 'https://bsky.app/profile/de1.pds.tentacle.expert/post/3l3w4tnezek2e',
+            'tags': 'count:1',
+            'subtitles': 'count:0',
+            'comments': list,
+        },
+    }]
+
+    def _get_comments(self, meta):
+        yield self.traverse_replies(meta, traverse_obj(meta, ('post', 'uri'), default=''))
```

Comment:
```suggestion
        yield from self.traverse_replies(meta, traverse_obj(meta, ('post', 'uri'), default=''))
```
Must be applied together with the suggestions on Line 219 and Line 198